client/apps/edit/public/icons/edit_section_remove.coffee

```diff
@@ -0,0 +1,25 @@
+React = require 'react'
+{ svg, circle, polygon } = React.DOM
+
+module.exports = React.createClass
+	displayName: 'EditSectionRemoveIcon'
+
+	render: ->
+  	svg {
```

Comment:
Couldn't get this running, seems to run into issues with the node globals here too -- ended up returning to the dangerously set html